lib/js/src/util/Version.js

```diff
@@ -98,7 +98,8 @@ class Version {
     */
     fromString(version) {
         let versions = version.split(".");
-        if (versions.length != 3) throw "Incorrect version string format";
+        if (versions.length != 3) 
+            throw "Incorrect version string format";
```

Comment:
Generally not a fan of this, but I'll allow it.

Reply:
Also, those quotes should get replaced by another linter PR.

Reply:
I am much more for adding { } if statement code paths

Reply:
> I am much more for adding { } around if statement code paths

Yes, I feel the same way. I'll check to see if there's a linter rule for that :)

Reply:
I'm not a fan of this either. Also for the throw. I have never done it this way and had thought it would work the same as a throw new Error(str), but I don't think this will give you the full stack trace so I feel this throw should throw a new Error.

Reply:
I agree with Benjamin on throwing strings and not errors. https://stackoverflow.com/questions/11502052/throwing-strings-instead-of-errors#comment80609827_11512623

Reply:
New rules to add to the linter:
https://eslint.org/docs/rules/curly
https://eslint.org/docs/rules/no-throw-literal